docker/Gemfile

```diff
@@ -2,7 +2,7 @@ source "https://rubygems.org"
 
 gem 'pupperware',
     :git => 'https://github.com/puppetlabs/pupperware.git',
-    :branch => 'master',
+    :branch => 'main',
```

Comment:
Took care of this one in https://github.com/puppetlabs/puppetdb/pull/3456

Reply:
Ok, thanks. I'll remove it.